app/views/shared/mno_offer_details/_giffgaff.html.erb

```diff
@@ -1,7 +1,12 @@
+<% brand = 'giffgaff' %>
+
 <ul class="govuk-list govuk-list--bullet">
   <li>The recipient will get 100GB of additional data, subject to giffgaff's <a href="https://www.giffgaff.com/terms/promotions#department_for_education_feb21">terms</a> and <a href="https://www.giffgaff.com/terms">fair use policy</a>.</li>
   <li>The offer is available to any eligible member with an active giffgaff goodybag.</li>
   <li>The offer is not available for pay as you go customers.</li>
   <li>A text message will be sent to the nominated device once the additional data has been added to the account.</li>
   <li>giffgaff will aim to process the request within 14 days.</li>
+  <% if MobileNetwork.excluded_fe_networks.pluck(:brand).include?(brand) %>
```

Comment:
`MobileNetwork.excluded_fe_networks.where(brand: brand).exists?` is a bit more efficient

Reply:
probably by a fraction